// @aws-sdk/client-connect/dist-es/models/enums.js
export const AccessType = {
    ALLOW: "ALLOW",
};
export const ActionType = {
    ASSIGN_CONTACT_CATEGORY: "ASSIGN_CONTACT_CATEGORY",
    ASSIGN_SLA: "ASSIGN_SLA",
    CREATE_CASE: "CREATE_CASE",
    CREATE_TASK: "CREATE_TASK",
    END_ASSOCIATED_TASKS: "END_ASSOCIATED_TASKS",
    GENERATE_EVENTBRIDGE_EVENT: "GENERATE_EVENTBRIDGE_EVENT",
    SEND_NOTIFICATION: "SEND_NOTIFICATION",
    SUBMIT_AUTO_EVALUATION: "SUBMIT_AUTO_EVALUATION",
    UPDATE_CASE: "UPDATE_CASE",
};
export const AgentAvailabilityTimer = {
    TIME_SINCE_LAST_ACTIVITY: "TIME_SINCE_LAST_ACTIVITY",
    TIME_SINCE_LAST_INBOUND: "TIME_SINCE_LAST_INBOUND",
};
export const ContactState = {
    CONNECTED: "CONNECTED",
    CONNECTED_ONHOLD: "CONNECTED_ONHOLD",
    CONNECTING: "CONNECTING",
    ENDED: "ENDED",
    ERROR: "ERROR",
    INCOMING: "INCOMING",
    MISSED: "MISSED",
    PENDING: "PENDING",
    REJECTED: "REJECTED",
};
export const Channel = {
    CHAT: "CHAT",
    EMAIL: "EMAIL",
    TASK: "TASK",
    VOICE: "VOICE",
};
export const ContactInitiationMethod = {
    AGENT_REPLY: "AGENT_REPLY",
    API: "API",
    CALLBACK: "CALLBACK",
    DISCONNECT: "DISCONNECT",
    EXTERNAL_OUTBOUND: "EXTERNAL_OUTBOUND",
    FLOW: "FLOW",
    INBOUND: "INBOUND",
    MONITOR: "MONITOR",
    OUTBOUND: "OUTBOUND",
    QUEUE_TRANSFER: "QUEUE_TRANSFER",
    TRANSFER: "TRANSFER",
    WEBRTC_API: "WEBRTC_API",
};
export const AllowedUserAction = {
    CALL: "CALL",
    DISCARD: "DISCARD",
};
export const ScreenShareCapability = {
    SEND: "SEND",
};
export const VideoCapability = {
    SEND: "SEND",
};
export const ParticipantState = {
    CONNECTED: "CONNECTED",
    DISCONNECTED: "DISCONNECTED",
    INITIAL: "INITIAL",
    MISSED: "MISSED",
};
export const AgentStatusState = {
    DISABLED: "DISABLED",
    ENABLED: "ENABLED",
};
export const AgentStatusType = {
    CUSTOM: "CUSTOM",
    OFFLINE: "OFFLINE",
    ROUTABLE: "ROUTABLE",
};
export const StringComparisonType = {
    CONTAINS: "CONTAINS",
    EXACT: "EXACT",
    STARTS_WITH: "STARTS_WITH",
};
export const AiUseCase = {
    AgentAssistance: "AgentAssistance",
    SelfService: "SelfService",
};
export const FlowModuleType = {
    MCP: "MCP",
};
export const MonitorCapability = {
    BARGE: "BARGE",
    SILENT_MONITOR: "SILENT_MONITOR",
};
export const AttachedFileInvalidRequestExceptionReason = {
    INVALID_FILE_NAME: "INVALID_FILE_NAME",
    INVALID_FILE_SIZE: "INVALID_FILE_SIZE",
    INVALID_FILE_TYPE: "INVALID_FILE_TYPE",
};
export const AttachedFileServiceQuotaExceededExceptionReason = {
    TOTAL_FILE_COUNT_EXCEEDED: "TOTAL_FILE_COUNT_EXCEEDED",
    TOTAL_FILE_SIZE_EXCEEDED: "TOTAL_FILE_SIZE_EXCEEDED",
};
export const VocabularyLanguageCode = {
    AR_AE: "ar-AE",
    CA_ES: "ca-ES",
    DA_DK: "da-DK",
    DE_CH: "de-CH",
    DE_DE: "de-DE",
    EN_AB: "en-AB",
    EN_AU: "en-AU",
    EN_GB: "en-GB",
    EN_IE: "en-IE",
    EN_IN: "en-IN",
    EN_NZ: "en-NZ",
    EN_US: "en-US",
    EN_WL: "en-WL",
    EN_ZA: "en-ZA",
    ES_ES: "es-ES",
    ES_US: "es-US",
    FI_FI: "fi-FI",
    FR_CA: "fr-CA",
    FR_FR: "fr-FR",
    HI_IN: "hi-IN",
    ID_ID: "id-ID",
    IT_IT: "it-IT",
    JA_JP: "ja-JP",
    KO_KR: "ko-KR",
    MS_MY: "ms-MY",
    NL_NL: "nl-NL",
    NO_NO: "no-NO",
    PL_PL: "pl-PL",
    PT_BR: "pt-BR",
    PT_PT: "pt-PT",
    SV_SE: "sv-SE",
    TL_PH: "tl-PH",
    ZH_CN: "zh-CN",
};
export const FlowAssociationResourceType = {
    ANALYTICS_CONNECTOR: "ANALYTICS_CONNECTOR",
    INBOUND_EMAIL: "INBOUND_EMAIL",
    OUTBOUND_EMAIL: "OUTBOUND_EMAIL",
    SMS_PHONE_NUMBER: "SMS_PHONE_NUMBER",
    WHATSAPP_MESSAGING_PHONE_NUMBER: "WHATSAPP_MESSAGING_PHONE_NUMBER",
};
export const InstanceStorageResourceType = {
    AGENT_EVENTS: "AGENT_EVENTS",
    ATTACHMENTS: "ATTACHMENTS",
    CALL_RECORDINGS: "CALL_RECORDINGS",
    CHAT_TRANSCRIPTS: "CHAT_TRANSCRIPTS",
    CONTACT_EVALUATIONS: "CONTACT_EVALUATIONS",
    CONTACT_TRACE_RECORDS: "CONTACT_TRACE_RECORDS",
    EMAIL_MESSAGES: "EMAIL_MESSAGES",
    MEDIA_STREAMS: "MEDIA_STREAMS",
    REAL_TIME_CONTACT_ANALYSIS_CHAT_SEGMENTS: "REAL_TIME_CONTACT_ANALYSIS_CHAT_SEGMENTS",
    REAL_TIME_CONTACT_ANALYSIS_SEGMENTS: "REAL_TIME_CONTACT_ANALYSIS_SEGMENTS",
    REAL_TIME_CONTACT_ANALYSIS_VOICE_SEGMENTS: "REAL_TIME_CONTACT_ANALYSIS_VOICE_SEGMENTS",
    SCHEDULED_REPORTS: "SCHEDULED_REPORTS",
    SCREEN_RECORDINGS: "SCREEN_RECORDINGS",
};
export const EncryptionType = {
    KMS: "KMS",
};
export const StorageType = {
    KINESIS_FIREHOSE: "KINESIS_FIREHOSE",
    KINESIS_STREAM: "KINESIS_STREAM",
    KINESIS_VIDEO_STREAM: "KINESIS_VIDEO_STREAM",
    S3: "S3",
};
export const EntityType = {
    AI_AGENT: "AI_AGENT",
    USER: "USER",
};
export const FileStatusType = {
    APPROVED: "APPROVED",
    FAILED: "FAILED",
    PROCESSING: "PROCESSING",
    REJECTED: "REJECTED",
};
export const FileUseCaseType = {
    ATTACHMENT: "ATTACHMENT",
    EMAIL_MESSAGE: "EMAIL_MESSAGE",
};
export const ListFlowAssociationResourceType = {
    ANALYTICS_CONNECTOR: "ANALYTICS_CONNECTOR",
    INBOUND_EMAIL: "INBOUND_EMAIL",
    OUTBOUND_EMAIL: "OUTBOUND_EMAIL",
    VOICE_PHONE_NUMBER: "VOICE_PHONE_NUMBER",
    WHATSAPP_MESSAGING_PHONE_NUMBER: "WHATSAPP_MESSAGING_PHONE_NUMBER",
};
export const EndpointType = {
    CONNECT_PHONENUMBER_ARN: "CONNECT_PHONENUMBER_ARN",
    CONTACT_FLOW: "CONTACT_FLOW",
    EMAIL_ADDRESS: "EMAIL_ADDRESS",
    TELEPHONE_NUMBER: "TELEPHONE_NUMBER",
    VOIP: "VOIP",
};
export const OutboundStrategyType = {
    AGENT_FIRST: "AGENT_FIRST",
};
export const FailureReasonCode = {
    IDEMPOTENCY_EXCEPTION: "IDEMPOTENCY_EXCEPTION",
    INTERNAL_ERROR: "INTERNAL_ERROR",
    INVALID_ATTRIBUTE_KEY: "INVALID_ATTRIBUTE_KEY",
    INVALID_CUSTOMER_ENDPOINT: "INVALID_CUSTOMER_ENDPOINT",
    INVALID_OUTBOUND_STRATEGY: "INVALID_OUTBOUND_STRATEGY",
    INVALID_QUEUE: "INVALID_QUEUE",
    INVALID_SYSTEM_ENDPOINT: "INVALID_SYSTEM_ENDPOINT",
    MISSING_CAMPAIGN: "MISSING_CAMPAIGN",
    MISSING_CUSTOMER_ENDPOINT: "MISSING_CUSTOMER_ENDPOINT",
    MISSING_QUEUE_ID_AND_SYSTEM_ENDPOINT: "MISSING_QUEUE_ID_AND_SYSTEM_ENDPOINT",
    REQUEST_THROTTLED: "REQUEST_THROTTLED",
};
export const InitiateAs = {
    COMPLETED: "COMPLETED",
    CONNECTED_TO_USER: "CONNECTED_TO_USER",
};
export const ReferenceStatus = {
    APPROVED: "APPROVED",
    AVAILABLE: "AVAILABLE",
    DELETED: "DELETED",
    FAILED: "FAILED",
    PROCESSING: "PROCESSING",
    REJECTED: "REJECTED",
};
export const ReferenceType = {
    ATTACHMENT: "ATTACHMENT",
    CONTACT_ANALYSIS: "CONTACT_ANALYSIS",
    DATE: "DATE",
    EMAIL: "EMAIL",
    EMAIL_MESSAGE: "EMAIL_MESSAGE",
    EMAIL_MESSAGE_PLAIN_TEXT: "EMAIL_MESSAGE_PLAIN_TEXT",
    NUMBER: "NUMBER",
    STRING: "STRING",
    URL: "URL",
};
export const ContactFlowStatus = {
    PUBLISHED: "PUBLISHED",
    SAVED: "SAVED",
};
export const ContactFlowType = {
    AGENT_HOLD: "AGENT_HOLD",
    AGENT_TRANSFER: "AGENT_TRANSFER",
    AGENT_WHISPER: "AGENT_WHISPER",
    CAMPAIGN: "CAMPAIGN",
    CONTACT_FLOW: "CONTACT_FLOW",
    CUSTOMER_HOLD: "CUSTOMER_HOLD",
    CUSTOMER_QUEUE: "CUSTOMER_QUEUE",
    CUSTOMER_WHISPER: "CUSTOMER_WHISPER",
    OUTBOUND_WHISPER: "OUTBOUND_WHISPER",
    QUEUE_TRANSFER: "QUEUE_TRANSFER",
};
export const DataTableStatus = {
    PUBLISHED: "PUBLISHED",
};
export const DataTableLockLevel = {
    ATTRIBUTE: "ATTRIBUTE",
    DATA_TABLE: "DATA_TABLE",
    NONE: "NONE",
    PRIMARY_VALUE: "PRIMARY_VALUE",
    VALUE: "VALUE",
};
export const DataTableAttributeValueType = {
    BOOLEAN: "BOOLEAN",
    NUMBER: "NUMBER",
    NUMBER_LIST: "NUMBER_LIST",
    TEXT: "TEXT",
    TEXT_LIST: "TEXT_LIST",
};
export const EvaluationFormItemEnablementAction = {
    DISABLE: "DISABLE",
    ENABLE: "ENABLE",
};
export const EvaluationFormItemSourceValuesComparator = {
    ALL_IN: "ALL_IN",
    EXACT: "EXACT",
    IN: "IN",
    NOT_IN: "NOT_IN",
};
export const EvaluationFormItemEnablementSourceType = {
    QUESTION_REF_ID: "QUESTION_REF_ID",
};
export const EvaluationFormItemEnablementSourceValueType = {
    OPTION_REF_ID: "OPTION_REF_ID",
};
export const EvaluationFormItemEnablementOperator = {
    AND: "AND",
    OR: "OR",
};
export const EvaluationFormQuestionType = {
    DATETIME: "DATETIME",
    MULTISELECT: "MULTISELECT",
    NUMERIC: "NUMERIC",
    SINGLESELECT: "SINGLESELECT",
    TEXT: "TEXT",
};
export const EvaluationFormQuestionAutomationAnswerSourceType = {
    CONTACT_LENS_DATA: "CONTACT_LENS_DATA",
    GEN_AI: "GEN_AI",
};
export const MultiSelectQuestionRuleCategoryAutomationCondition = {
    NOT_PRESENT: "NOT_PRESENT",
    PRESENT: "PRESENT",
};
export const EvaluationFormMultiSelectQuestionDisplayMode = {
    CHECKBOX: "CHECKBOX",
    DROPDOWN: "DROPDOWN",
};
export const NumericQuestionPropertyAutomationLabel = {
    AGENT_INTERACTION_AND_HOLD_DURATION: "AGENT_INTERACTION_AND_HOLD_DURATION",
    AGENT_INTERACTION_DURATION: "AGENT_INTERACTION_DURATION",
    CONTACT_DURATION: "CONTACT_DURATION",
    CUSTOMER_HOLD_TIME: "CUSTOMER_HOLD_TIME",
    CUSTOMER_SENTIMENT_SCORE_WITHOUT_AGENT: "CUSTOMER_SENTIMENT_SCORE_WITHOUT_AGENT",
    CUSTOMER_SENTIMENT_SCORE_WITH_AGENT: "CUSTOMER_SENTIMENT_SCORE_WITH_AGENT",
    LONGEST_HOLD_DURATION: "LONGEST_HOLD_DURATION",
    NON_TALK_TIME: "NON_TALK_TIME",
    NON_TALK_TIME_PERCENTAGE: "NON_TALK_TIME_PERCENTAGE",
    NUMBER_OF_HOLDS: "NUMBER_OF_HOLDS",
    NUMBER_OF_INTERRUPTIONS: "NUMBER_OF_INTERRUPTIONS",
    OVERALL_AGENT_SENTIMENT_SCORE: "OVERALL_AGENT_SENTIMENT_SCORE",
    OVERALL_CUSTOMER_SENTIMENT_SCORE: "OVERALL_CUSTOMER_SENTIMENT_SCORE",
};
export const SingleSelectQuestionRuleCategoryAutomationCondition = {
    NOT_PRESENT: "NOT_PRESENT",
    PRESENT: "PRESENT",
};
export const EvaluationFormSingleSelectQuestionDisplayMode = {
    DROPDOWN: "DROPDOWN",
    RADIO: "RADIO",
};
export const EvaluationFormLanguageCode = {
    DE_DE: "de-DE",
    EN_US: "en-US",
    ES_ES: "es-ES",
    FR_FR: "fr-FR",
    IT_IT: "it-IT",
    PT_BR: "pt-BR",
};
export const EvaluationReviewNotificationRecipientType = {
    USER_ID: "USER_ID",
};
export const EvaluationFormScoringMode = {
    QUESTION_ONLY: "QUESTION_ONLY",
    SECTION_ONLY: "SECTION_ONLY",
};
export const EvaluationFormScoringStatus = {
    DISABLED: "DISABLED",
    ENABLED: "ENABLED",
};
export const ContactInteractionType = {
    AGENT: "AGENT",
    AUTOMATED: "AUTOMATED",
};
export const HoursOfOperationDays = {
    FRIDAY: "FRIDAY",
    MONDAY: "MONDAY",
    SATURDAY: "SATURDAY",
    SUNDAY: "SUNDAY",
    THURSDAY: "THURSDAY",
    TUESDAY: "TUESDAY",
    WEDNESDAY: "WEDNESDAY",
};
export const OverrideDays = {
    FRIDAY: "FRIDAY",
    MONDAY: "MONDAY",
    SATURDAY: "SATURDAY",
    SUNDAY: "SUNDAY",
    THURSDAY: "THURSDAY",
    TUESDAY: "TUESDAY",
    WEDNESDAY: "WEDNESDAY",
};
export const OverrideType = {
    CLOSED: "CLOSED",
    OPEN: "OPEN",
    STANDARD: "STANDARD",
};
export const RecurrenceFrequency = {
    MONTHLY: "MONTHLY",
    WEEKLY: "WEEKLY",
    YEARLY: "YEARLY",
};
export const DirectoryType = {
    CONNECT_MANAGED: "CONNECT_MANAGED",
    EXISTING_DIRECTORY: "EXISTING_DIRECTORY",
    SAML: "SAML",
};
export const IntegrationType = {
    ANALYTICS_CONNECTOR: "ANALYTICS_CONNECTOR",
    APPLICATION: "APPLICATION",
    CALL_TRANSFER_CONNECTOR: "CALL_TRANSFER_CONNECTOR",
    CASES_DOMAIN: "CASES_DOMAIN",
    COGNITO_USER_POOL: "COGNITO_USER_POOL",
    EVENT: "EVENT",
    FILE_SCANNER: "FILE_SCANNER",
    MESSAGE_PROCESSOR: "MESSAGE_PROCESSOR",
    PINPOINT_APP: "PINPOINT_APP",
    Q_MESSAGE_TEMPLATES: "Q_MESSAGE_TEMPLATES",
    SES_IDENTITY: "SES_IDENTITY",
    VOICE_ID: "VOICE_ID",
    WISDOM_ASSISTANT: "WISDOM_ASSISTANT",
    WISDOM_KNOWLEDGE_BASE: "WISDOM_KNOWLEDGE_BASE",
    WISDOM_QUICK_RESPONSES: "WISDOM_QUICK_RESPONSES",
};
export const SourceType = {
    CASES: "CASES",
    SALESFORCE: "SALESFORCE",
    ZENDESK: "ZENDESK",
};
export const ParticipantRole = {
    AGENT: "AGENT",
    CUSTOMER: "CUSTOMER",
    CUSTOM_BOT: "CUSTOM_BOT",
    SUPERVISOR: "SUPERVISOR",
    SYSTEM: "SYSTEM",
};
export const RehydrationType = {
    ENTIRE_PAST_SESSION: "ENTIRE_PAST_SESSION",
    FROM_SEGMENT: "FROM_SEGMENT",
};
export const DeviceType = {
    APNS: "APNS",
    APNS_SANDBOX: "APNS_SANDBOX",
    GCM: "GCM",
};
export const QuickConnectType = {
    FLOW: "FLOW",
    PHONE_NUMBER: "PHONE_NUMBER",
    QUEUE: "QUEUE",
    USER: "USER",
};
export const BehaviorType = {
    ROUTE_ANY_CHANNEL: "ROUTE_ANY_CHANNEL",
    ROUTE_CURRENT_CHANNEL_ONLY: "ROUTE_CURRENT_CHANNEL_ONLY",
};
export const SlaType = {
    CASE_FIELD: "CaseField",
};
export const SlaAssignmentType = {
    CASES: "CASES",
};
export const NotificationContentType = {
    PLAIN_TEXT: "PLAIN_TEXT",
};
export const NotificationDeliveryType = {
    EMAIL: "EMAIL",
};
export const RulePublishStatus = {
    DRAFT: "DRAFT",
    PUBLISHED: "PUBLISHED",
};
export const EventSourceName = {
    OnCaseCreate: "OnCaseCreate",
    OnCaseUpdate: "OnCaseUpdate",
    OnContactEvaluationSubmit: "OnContactEvaluationSubmit",
    OnMetricDataUpdate: "OnMetricDataUpdate",
    OnPostCallAnalysisAvailable: "OnPostCallAnalysisAvailable",
    OnPostChatAnalysisAvailable: "OnPostChatAnalysisAvailable",
    OnRealTimeCallAnalysisAvailable: "OnRealTimeCallAnalysisAvailable",
    OnRealTimeChatAnalysisAvailable: "OnRealTimeChatAnalysisAvailable",
    OnSalesforceCaseCreate: "OnSalesforceCaseCreate",
    OnSlaBreach: "OnSlaBreach",
    OnZendeskTicketCreate: "OnZendeskTicketCreate",
    OnZendeskTicketStatusUpdate: "OnZendeskTicketStatusUpdate",
};
export const ApplicationType = {
    MCP: "MCP",
    THIRD_PARTY_APPLICATION: "THIRD_PARTY_APPLICATION",
};
export const TaskTemplateFieldType = {
    BOOLEAN: "BOOLEAN",
    DATE_TIME: "DATE_TIME",
    DESCRIPTION: "DESCRIPTION",
    EMAIL: "EMAIL",
    EXPIRY_DURATION: "EXPIRY_DURATION",
    NAME: "NAME",
    NUMBER: "NUMBER",
    QUICK_CONNECT: "QUICK_CONNECT",
    SCHEDULED_TIME: "SCHEDULED_TIME",
    SELF_ASSIGN: "SELF_ASSIGN",
    SINGLE_SELECT: "SINGLE_SELECT",
    TEXT: "TEXT",
    TEXT_AREA: "TEXT_AREA",
    URL: "URL",
};
export const TaskTemplateStatus = {
    ACTIVE: "ACTIVE",
    INACTIVE: "INACTIVE",
};
export const PropertyValidationExceptionReason = {
    INVALID_FORMAT: "INVALID_FORMAT",
    NOT_SUPPORTED: "NOT_SUPPORTED",
    REFERENCED_RESOURCE_NOT_FOUND: "REFERENCED_RESOURCE_NOT_FOUND",
    REQUIRED_PROPERTY_MISSING: "REQUIRED_PROPERTY_MISSING",
    RESOURCE_NAME_ALREADY_EXISTS: "RESOURCE_NAME_ALREADY_EXISTS",
    UNIQUE_CONSTRAINT_VIOLATED: "UNIQUE_CONSTRAINT_VIOLATED",
};
export const UseCaseType = {
    CONNECT_CAMPAIGNS: "CONNECT_CAMPAIGNS",
    RULES_EVALUATION: "RULES_EVALUATION",
};
export const PhoneType = {
    DESK_PHONE: "DESK_PHONE",
    SOFT_PHONE: "SOFT_PHONE",
};
export const ViewStatus = {
    PUBLISHED: "PUBLISHED",
    SAVED: "SAVED",
};
export const ViewType = {
    AWS_MANAGED: "AWS_MANAGED",
    CUSTOMER_MANAGED: "CUSTOMER_MANAGED",
};
export const ResourceType = {
    CONTACT: "CONTACT",
    CONTACT_FLOW: "CONTACT_FLOW",
    HIERARCHY_GROUP: "HIERARCHY_GROUP",
    HIERARCHY_LEVEL: "HIERARCHY_LEVEL",
    INSTANCE: "INSTANCE",
    PARTICIPANT: "PARTICIPANT",
    PHONE_NUMBER: "PHONE_NUMBER",
    USER: "USER",
};
export const VocabularyState = {
    ACTIVE: "ACTIVE",
    CREATION_FAILED: "CREATION_FAILED",
    CREATION_IN_PROGRESS: "CREATION_IN_PROGRESS",
    DELETE_IN_PROGRESS: "DELETE_IN_PROGRESS",
};
export const WorkspaceFontFamily = {
    ARIAL: "Arial",
    COURIER_NEW: "Courier New",
    GEORGIA: "Georgia",
    TIMES_NEW_ROMAN: "Times New Roman",
    TREBUCHET: "Trebuchet",
    VERDANA: "Verdana",
};
export const MediaType = {
    IMAGE_LOGO_DARK_FAVICON: "IMAGE_LOGO_DARK_FAVICON",
    IMAGE_LOGO_DARK_HORIZONTAL: "IMAGE_LOGO_DARK_HORIZONTAL",
    IMAGE_LOGO_LIGHT_FAVICON: "IMAGE_LOGO_LIGHT_FAVICON",
    IMAGE_LOGO_LIGHT_HORIZONTAL: "IMAGE_LOGO_LIGHT_HORIZONTAL",
};
export const AnsweringMachineDetectionStatus = {
    AMD_ERROR: "AMD_ERROR",
    AMD_NOT_APPLICABLE: "AMD_NOT_APPLICABLE",
    AMD_UNANSWERED: "AMD_UNANSWERED",
    AMD_UNRESOLVED: "AMD_UNRESOLVED",
    ANSWERED: "ANSWERED",
    ERROR: "ERROR",
    FAX_MACHINE_DETECTED: "FAX_MACHINE_DETECTED",
    HUMAN_ANSWERED: "HUMAN_ANSWERED",
    SIT_TONE_BUSY: "SIT_TONE_BUSY",
    SIT_TONE_DETECTED: "SIT_TONE_DETECTED",
    SIT_TONE_INVALID_NUMBER: "SIT_TONE_INVALID_NUMBER",
    UNDETECTED: "UNDETECTED",
    VOICEMAIL_BEEP: "VOICEMAIL_BEEP",
    VOICEMAIL_NO_BEEP: "VOICEMAIL_NO_BEEP",
};
export const ParticipantType = {
    AGENT: "AGENT",
    ALL: "ALL",
    CUSTOMER: "CUSTOMER",
    MANAGER: "MANAGER",
    THIRDPARTY: "THIRDPARTY",
};
export const Status = {
    COMPLETE: "COMPLETE",
    DELETED: "DELETED",
    IN_PROGRESS: "IN_PROGRESS",
};
export const NextContactType = {
    QUICK_CONNECT: "QUICK_CONNECT",
};
export const MediaStreamType = {
    AUDIO: "AUDIO",
    VIDEO: "VIDEO",
};
export const RecordingStatus = {
    AVAILABLE: "AVAILABLE",
    DELETED: "DELETED",
};
export const RoutingCriteriaStepStatus = {
    ACTIVE: "ACTIVE",
    EXPIRED: "EXPIRED",
    INACTIVE: "INACTIVE",
    JOINED: "JOINED",
};
export const QuestionRuleCategoryAutomationCondition = {
    NOT_PRESENT: "NOT_PRESENT",
    PRESENT: "PRESENT",
};
export const EvaluationQuestionAnswerAnalysisType = {
    CONTACT_LENS_DATA: "CONTACT_LENS_DATA",
    GEN_AI: "GEN_AI",
};
export const EvaluationTranscriptType = {
    RAW: "RAW",
    REDACTED: "REDACTED",
};
export const EvaluationSuggestedAnswerStatus = {
    FAILED: "FAILED",
    IN_PROGRESS: "IN_PROGRESS",
    SUCCEEDED: "SUCCEEDED",
};
export const EvaluationType = {
    CALIBRATION: "CALIBRATION",
    STANDARD: "STANDARD",
};
export const AutoEvaluationStatus = {
    FAILED: "FAILED",
    IN_PROGRESS: "IN_PROGRESS",
    SUCCEEDED: "SUCCEEDED",
};
export const ContactParticipantRole = {
    AGENT: "AGENT",
    CUSTOM_BOT: "CUSTOM_BOT",
    SYSTEM: "SYSTEM",
};
export const EvaluationStatus = {
    DRAFT: "DRAFT",
    REVIEW_REQUESTED: "REVIEW_REQUESTED",
    SUBMITTED: "SUBMITTED",
    UNDER_REVIEW: "UNDER_REVIEW",
};
export const ContactFlowState = {
    ACTIVE: "ACTIVE",
    ARCHIVED: "ARCHIVED",
};
export const ContactFlowModuleState = {
    ACTIVE: "ACTIVE",
    ARCHIVED: "ARCHIVED",
};
export const ContactFlowModuleStatus = {
    PUBLISHED: "PUBLISHED",
    SAVED: "SAVED",
};
export const EvaluationFormVersionStatus = {
    ACTIVE: "ACTIVE",
    DRAFT: "DRAFT",
};
export const InstanceStatus = {
    ACTIVE: "ACTIVE",
    CREATION_FAILED: "CREATION_FAILED",
    CREATION_IN_PROGRESS: "CREATION_IN_PROGRESS",
};
export const InstanceReplicationStatus = {
    INSTANCE_REPLICATION_COMPLETE: "INSTANCE_REPLICATION_COMPLETE",
    INSTANCE_REPLICATION_DELETION_FAILED: "INSTANCE_REPLICATION_DELETION_FAILED",
    INSTANCE_REPLICATION_FAILED: "INSTANCE_REPLICATION_FAILED",
    INSTANCE_REPLICATION_IN_PROGRESS: "INSTANCE_REPLICATION_IN_PROGRESS",
    INSTANCE_REPLICA_DELETING: "INSTANCE_REPLICA_DELETING",
    RESOURCE_REPLICATION_NOT_STARTED: "RESOURCE_REPLICATION_NOT_STARTED",
};
export const InstanceAttributeType = {
    AUTO_RESOLVE_BEST_VOICES: "AUTO_RESOLVE_BEST_VOICES",
    CONTACTFLOW_LOGS: "CONTACTFLOW_LOGS",
    CONTACT_LENS: "CONTACT_LENS",
    EARLY_MEDIA: "EARLY_MEDIA",
    ENHANCED_CHAT_MONITORING: "ENHANCED_CHAT_MONITORING",
    ENHANCED_CONTACT_MONITORING: "ENHANCED_CONTACT_MONITORING",
    HIGH_VOLUME_OUTBOUND: "HIGH_VOLUME_OUTBOUND",
    INBOUND_CALLS: "INBOUND_CALLS",
    MESSAGE_STREAMING: "MESSAGE_STREAMING",
    MULTI_PARTY_CHAT_CONFERENCE: "MULTI_PARTY_CHAT_CONFERENCE",
    MULTI_PARTY_CONFERENCE: "MULTI_PARTY_CONFERENCE",
    OUTBOUND_CALLS: "OUTBOUND_CALLS",
    USE_CUSTOM_TTS_VOICES: "USE_CUSTOM_TTS_VOICES",
};
export const PhoneNumberCountryCode = {
    AD: "AD",
    AE: "AE",
    AF: "AF",
    AG: "AG",
    AI: "AI",
    AL: "AL",
    AM: "AM",
    AN: "AN",
    AO: "AO",
    AQ: "AQ",
    AR: "AR",
    AS: "AS",
    AT: "AT",
    AU: "AU",
    AW: "AW",
    AZ: "AZ",
    BA: "BA",
    BB: "BB",
    BD: "BD",
    BE: "BE",
    BF: "BF",
    BG: "BG",
    BH: "BH",
    BI: "BI",
    BJ: "BJ",
    BL: "BL",
    BM: "BM",
    BN: "BN",
    BO: "BO",
    BR: "BR",
    BS: "BS",
    BT: "BT",
    BW: "BW",
    BY: "BY",
    BZ: "BZ",
    CA: "CA",
    CC: "CC",
    CD: "CD",
    CF: "CF",
    CG: "CG",
    CH: "CH",
    CI: "CI",
    CK: "CK",
    CL: "CL",
    CM: "CM",
    CN: "CN",
    CO: "CO",
    CR: "CR",
    CU: "CU",
    CV: "CV",
    CW: "CW",
    CX: "CX",
    CY: "CY",
    CZ: "CZ",
    DE: "DE",
    DJ: "DJ",
    DK: "DK",
    DM: "DM",
    DO: "DO",
    DZ: "DZ",
    EC: "EC",
    EE: "EE",
    EG: "EG",
    EH: "EH",
    ER: "ER",
    ES: "ES",
    ET: "ET",
    FI: "FI",
    FJ: "FJ",
    FK: "FK",
    FM: "FM",
    FO: "FO",
    FR: "FR",
    GA: "GA",
    GB: "GB",
    GD: "GD",
    GE: "GE",
    GG: "GG",
    GH: "GH",
    GI: "GI",
    GL: "GL",
    GM: "GM",
    GN: "GN",
    GQ: "GQ",
    GR: "GR",
    GT: "GT",
    GU: "GU",
    GW: "GW",
    GY: "GY",
    HK: "HK",
    HN: "HN",
    HR: "HR",
    HT: "HT",
    HU: "HU",
    ID: "ID",
    IE: "IE",
    IL: "IL",
    IM: "IM",
    IN: "IN",
    IO: "IO",
    IQ: "IQ",
    IR: "IR",
    IS: "IS",
    IT: "IT",
    JE: "JE",
    JM: "JM",
    JO: "JO",
    JP: "JP",
    KE: "KE",
    KG: "KG",
    KH: "KH",
    KI: "KI",
    KM: "KM",
    KN: "KN",
    KP: "KP",
    KR: "KR",
    KW: "KW",
    KY: "KY",
    KZ: "KZ",
    LA: "LA",
    LB: "LB",
    LC: "LC",
    LI: "LI",
    LK: "LK",
    LR: "LR",
    LS: "LS",
    LT: "LT",
    LU: "LU",
    LV: "LV",
    LY: "LY",
    MA: "MA",
    MC: "MC",
    MD: "MD",
    ME: "ME",
    MF: "MF",
    MG: "MG",
    MH: "MH",
    MK: "MK",
    ML: "ML",
    MM: "MM",
    MN: "MN",
    MO: "MO",
    MP: "MP",
    MR: "MR",
    MS: "MS",
    MT: "MT",
    MU: "MU",
    MV: "MV",
    MW: "MW",
    MX: "MX",
    MY: "MY",
    MZ: "MZ",
    NA: "NA",
    NC: "NC",
    NE: "NE",
    NG: "NG",
    NI: "NI",
    NL: "NL",
    NO: "NO",
    NP: "NP",
    NR: "NR",
    NU: "NU",
    NZ: "NZ",
    OM: "OM",
    PA: "PA",
    PE: "PE",
    PF: "PF",
    PG: "PG",
    PH: "PH",
    PK: "PK",
    PL: "PL",
    PM: "PM",
    PN: "PN",
    PR: "PR",
    PT: "PT",
    PW: "PW",
    PY: "PY",
    QA: "QA",
    RE: "RE",
    RO: "RO",
    RS: "RS",
    RU: "RU",
    RW: "RW",
    SA: "SA",
    SB: "SB",
    SC: "SC",
    SD: "SD",
    SE: "SE",
    SG: "SG",
    SH: "SH",
    SI: "SI",
    SJ: "SJ",
    SK: "SK",
    SL: "SL",
    SM: "SM",
    SN: "SN",
    SO: "SO",
    SR: "SR",
    ST: "ST",
    SV: "SV",
    SX: "SX",
    SY: "SY",
    SZ: "SZ",
    TC: "TC",
    TD: "TD",
    TG: "TG",
    TH: "TH",
    TJ: "TJ",
    TK: "TK",
    TL: "TL",
    TM: "TM",
    TN: "TN",
    TO: "TO",
    TR: "TR",
    TT: "TT",
    TV: "TV",
    TW: "TW",
    TZ: "TZ",
    UA: "UA",
    UG: "UG",
    US: "US",
    UY: "UY",
    UZ: "UZ",
    VA: "VA",
    VC: "VC",
    VE: "VE",
    VG: "VG",
    VI: "VI",
    VN: "VN",
    VU: "VU",
    WF: "WF",
    WS: "WS",
    YE: "YE",
    YT: "YT",
    ZA: "ZA",
    ZM: "ZM",
    ZW: "ZW",
};
export const PhoneNumberWorkflowStatus = {
    Claimed: "CLAIMED",
    Failed: "FAILED",
    InProgress: "IN_PROGRESS",
};
export const PhoneNumberType = {
    DID: "DID",
    SHARED: "SHARED",
    SHORT_CODE: "SHORT_CODE",
    THIRD_PARTY_DID: "THIRD_PARTY_DID",
    THIRD_PARTY_TF: "THIRD_PARTY_TF",
    TOLL_FREE: "TOLL_FREE",
    UIFN: "UIFN",
};
export const QueueStatus = {
    DISABLED: "DISABLED",
    ENABLED: "ENABLED",
};
export const TrafficDistributionGroupStatus = {
    ACTIVE: "ACTIVE",
    CREATION_FAILED: "CREATION_FAILED",
    CREATION_IN_PROGRESS: "CREATION_IN_PROGRESS",
    DELETION_FAILED: "DELETION_FAILED",
    PENDING_DELETION: "PENDING_DELETION",
    UPDATE_IN_PROGRESS: "UPDATE_IN_PROGRESS",
};
export const Visibility = {
    All: "ALL",
    Assigned: "ASSIGNED",
    None: "NONE",
};
export const ContactMetricName = {
    POSITION_IN_QUEUE: "POSITION_IN_QUEUE",
};
export const CurrentMetricName = {
    AGENTS_AFTER_CONTACT_WORK: "AGENTS_AFTER_CONTACT_WORK",
    AGENTS_AVAILABLE: "AGENTS_AVAILABLE",
    AGENTS_ERROR: "AGENTS_ERROR",
    AGENTS_NON_PRODUCTIVE: "AGENTS_NON_PRODUCTIVE",
    AGENTS_ONLINE: "AGENTS_ONLINE",
    AGENTS_ON_CALL: "AGENTS_ON_CALL",
    AGENTS_ON_CONTACT: "AGENTS_ON_CONTACT",
    AGENTS_STAFFED: "AGENTS_STAFFED",
    CONTACTS_IN_QUEUE: "CONTACTS_IN_QUEUE",
    CONTACTS_SCHEDULED: "CONTACTS_SCHEDULED",
    OLDEST_CONTACT_AGE: "OLDEST_CONTACT_AGE",
    SLOTS_ACTIVE: "SLOTS_ACTIVE",
    SLOTS_AVAILABLE: "SLOTS_AVAILABLE",
};
export const Unit = {
    COUNT: "COUNT",
    PERCENT: "PERCENT",
    SECONDS: "SECONDS",
};
export const Grouping = {
    AGENT_STATUS: "AGENT_STATUS",
    CHANNEL: "CHANNEL",
    QUEUE: "QUEUE",
    ROUTING_PROFILE: "ROUTING_PROFILE",
    ROUTING_STEP_EXPRESSION: "ROUTING_STEP_EXPRESSION",
    SUBTYPE: "SUBTYPE",
    VALIDATION_TEST_TYPE: "VALIDATION_TEST_TYPE",
};
export const SortOrder = {
    ASCENDING: "ASCENDING",
    DESCENDING: "DESCENDING",
};
export const OperationalStatus = {
    CLOSED: "CLOSED",
    OPEN: "OPEN",
};
export const HistoricalMetricName = {
    ABANDON_TIME: "ABANDON_TIME",
    AFTER_CONTACT_WORK_TIME: "AFTER_CONTACT_WORK_TIME",
    API_CONTACTS_HANDLED: "API_CONTACTS_HANDLED",
    CALLBACK_CONTACTS_HANDLED: "CALLBACK_CONTACTS_HANDLED",
    CONTACTS_ABANDONED: "CONTACTS_ABANDONED",
    CONTACTS_AGENT_HUNG_UP_FIRST: "CONTACTS_AGENT_HUNG_UP_FIRST",
    CONTACTS_CONSULTED: "CONTACTS_CONSULTED",
    CONTACTS_HANDLED: "CONTACTS_HANDLED",
    CONTACTS_HANDLED_INCOMING: "CONTACTS_HANDLED_INCOMING",
    CONTACTS_HANDLED_OUTBOUND: "CONTACTS_HANDLED_OUTBOUND",
    CONTACTS_HOLD_ABANDONS: "CONTACTS_HOLD_ABANDONS",
    CONTACTS_MISSED: "CONTACTS_MISSED",
    CONTACTS_QUEUED: "CONTACTS_QUEUED",
    CONTACTS_TRANSFERRED_IN: "CONTACTS_TRANSFERRED_IN",
    CONTACTS_TRANSFERRED_IN_FROM_QUEUE: "CONTACTS_TRANSFERRED_IN_FROM_QUEUE",
    CONTACTS_TRANSFERRED_OUT: "CONTACTS_TRANSFERRED_OUT",
    CONTACTS_TRANSFERRED_OUT_FROM_QUEUE: "CONTACTS_TRANSFERRED_OUT_FROM_QUEUE",
    HANDLE_TIME: "HANDLE_TIME",
    HOLD_TIME: "HOLD_TIME",
    INTERACTION_AND_HOLD_TIME: "INTERACTION_AND_HOLD_TIME",
    INTERACTION_TIME: "INTERACTION_TIME",
    OCCUPANCY: "OCCUPANCY",
    QUEUED_TIME: "QUEUED_TIME",
    QUEUE_ANSWER_TIME: "QUEUE_ANSWER_TIME",
    SERVICE_LEVEL: "SERVICE_LEVEL",
};
export const Statistic = {
    AVG: "AVG",
    MAX: "MAX",
    SUM: "SUM",
};
export const Comparison = {
    LT: "LT",
};
export const FilterV2StringConditionComparisonOperator = {
    NOT_EXISTS: "NOT_EXISTS",
};
export const IntervalPeriod = {
    DAY: "DAY",
    FIFTEEN_MIN: "FIFTEEN_MIN",
    HOUR: "HOUR",
    THIRTY_MIN: "THIRTY_MIN",
    TOTAL: "TOTAL",
    WEEK: "WEEK",
};
export const LexVersion = {
    V1: "V1",
    V2: "V2",
};
export const QueueType = {
    AGENT: "AGENT",
    STANDARD: "STANDARD",
};
export const RealTimeContactAnalysisOutputType = {
    Raw: "Raw",
    Redacted: "Redacted",
};
export const RealTimeContactAnalysisSegmentType = {
    Attachments: "Attachments",
    Categories: "Categories",
    Event: "Event",
    Issues: "Issues",
    PostContactSummary: "PostContactSummary",
    Transcript: "Transcript",
};
export const RealTimeContactAnalysisSupportedChannel = {
    CHAT: "CHAT",
    VOICE: "VOICE",
};
export const ArtifactStatus = {
    APPROVED: "APPROVED",
    IN_PROGRESS: "IN_PROGRESS",
    REJECTED: "REJECTED",
};
export const RealTimeContactAnalysisPostContactSummaryFailureCode = {
    FAILED_SAFETY_GUIDELINES: "FAILED_SAFETY_GUIDELINES",
    INSUFFICIENT_CONVERSATION_CONTENT: "INSUFFICIENT_CONVERSATION_CONTENT",
    INTERNAL_ERROR: "INTERNAL_ERROR",
    INVALID_ANALYSIS_CONFIGURATION: "INVALID_ANALYSIS_CONFIGURATION",
    QUOTA_EXCEEDED: "QUOTA_EXCEEDED",
};
export const RealTimeContactAnalysisPostContactSummaryStatus = {
    COMPLETED: "COMPLETED",
    FAILED: "FAILED",
};
export const RealTimeContactAnalysisSentimentLabel = {
    NEGATIVE: "NEGATIVE",
    NEUTRAL: "NEUTRAL",
    POSITIVE: "POSITIVE",
};
export const RealTimeContactAnalysisStatus = {
    COMPLETED: "COMPLETED",
    FAILED: "FAILED",
    IN_PROGRESS: "IN_PROGRESS",
};
export const ContactRecordingType = {
    AGENT: "AGENT",
    IVR: "IVR",
    SCREEN: "SCREEN",
};
export const BooleanComparisonType = {
    IS_FALSE: "IS_FALSE",
    IS_TRUE: "IS_TRUE",
};
export const DateTimeComparisonType = {
    EQUAL_TO: "EQUAL_TO",
    GREATER_THAN: "GREATER_THAN",
    GREATER_THAN_OR_EQUAL_TO: "GREATER_THAN_OR_EQUAL_TO",
    LESS_THAN: "LESS_THAN",
    LESS_THAN_OR_EQUAL_TO: "LESS_THAN_OR_EQUAL_TO",
    RANGE: "RANGE",
};
export const DecimalComparisonType = {
    EQUAL: "EQUAL",
    GREATER: "GREATER",
    GREATER_OR_EQUAL: "GREATER_OR_EQUAL",
    LESSER: "LESSER",
    LESSER_OR_EQUAL: "LESSER_OR_EQUAL",
    NOT_EQUAL: "NOT_EQUAL",
    RANGE: "RANGE",
};
export const NumberComparisonType = {
    EQUAL: "EQUAL",
    GREATER: "GREATER",
    GREATER_OR_EQUAL: "GREATER_OR_EQUAL",
    LESSER: "LESSER",
    LESSER_OR_EQUAL: "LESSER_OR_EQUAL",
    NOT_EQUAL: "NOT_EQUAL",
    RANGE: "RANGE",
};
export const SearchContactsTimeRangeType = {
    CONNECTED_TO_AGENT_TIMESTAMP: "CONNECTED_TO_AGENT_TIMESTAMP",
    DISCONNECT_TIMESTAMP: "DISCONNECT_TIMESTAMP",
    ENQUEUE_TIMESTAMP: "ENQUEUE_TIMESTAMP",
    INITIATION_TIMESTAMP: "INITIATION_TIMESTAMP",
    SCHEDULED_TIMESTAMP: "SCHEDULED_TIMESTAMP",
};
export const SearchContactsTimeRangeConditionType = {
    NOT_EXISTS: "NOT_EXISTS",
};
export const SearchContactsMatchType = {
    MATCH_ALL: "MATCH_ALL",
    MATCH_ANY: "MATCH_ANY",
    MATCH_EXACT: "MATCH_EXACT",
    MATCH_NONE: "MATCH_NONE",
};
export const SortableFieldName = {
    CHANNEL: "CHANNEL",
    CONNECTED_TO_AGENT_TIMESTAMP: "CONNECTED_TO_AGENT_TIMESTAMP",
    DISCONNECT_TIMESTAMP: "DISCONNECT_TIMESTAMP",
    EXPIRY_TIMESTAMP: "EXPIRY_TIMESTAMP",
    INITIATION_METHOD: "INITIATION_METHOD",
    INITIATION_TIMESTAMP: "INITIATION_TIMESTAMP",
    SCHEDULED_TIMESTAMP: "SCHEDULED_TIMESTAMP",
};
export const DateComparisonType = {
    EQUAL_TO: "EQUAL_TO",
    GREATER_THAN: "GREATER_THAN",
    GREATER_THAN_OR_EQUAL_TO: "GREATER_THAN_OR_EQUAL_TO",
    LESS_THAN: "LESS_THAN",
    LESS_THAN_OR_EQUAL_TO: "LESS_THAN_OR_EQUAL_TO",
};
export const SearchableQueueType = {
    STANDARD: "STANDARD",
};
export const HierarchyGroupMatchType = {
    EXACT: "EXACT",
    WITH_CHILD_GROUPS: "WITH_CHILD_GROUPS",
};
export const TargetListType = {
    PROFICIENCIES: "PROFICIENCIES",
};
export const ChatEventType = {
    DISCONNECT: "DISCONNECT",
    EVENT: "EVENT",
    MESSAGE: "MESSAGE",
};
export const OutboundMessageSourceType = {
    RAW: "RAW",
    TEMPLATE: "TEMPLATE",
};
export const TrafficType = {
    CAMPAIGN: "CAMPAIGN",
    GENERAL: "GENERAL",
};
export const DisconnectOnCustomerExitParticipantType = {
    AGENT: "AGENT",
};
export const ResponseMode = {
    COMPLETE: "COMPLETE",
    INCREMENTAL: "INCREMENTAL",
};
export const ContactMediaProcessingFailureMode = {
    DELIVER_UNPROCESSED_MESSAGE: "DELIVER_UNPROCESSED_MESSAGE",
    DO_NOT_DELIVER_UNPROCESSED_MESSAGE: "DO_NOT_DELIVER_UNPROCESSED_MESSAGE",
};
export const IvrRecordingTrack = {
    ALL: "ALL",
};
export const VoiceRecordingTrack = {
    ALL: "ALL",
    FROM_AGENT: "FROM_AGENT",
    TO_AGENT: "TO_AGENT",
};
export const InboundMessageSourceType = {
    RAW: "RAW",
};
export const EmailHeaderType = {
    IN_REPLY_TO: "IN_REPLY_TO",
    MESSAGE_ID: "MESSAGE_ID",
    REFERENCES: "REFERENCES",
    X_SES_SPAM_VERDICT: "X_SES_SPAM_VERDICT",
    X_SES_VIRUS_VERDICT: "X_SES_VIRUS_VERDICT",
};
export const MeetingFeatureStatus = {
    AVAILABLE: "AVAILABLE",
    UNAVAILABLE: "UNAVAILABLE",
};
export const TimerEligibleParticipantRoles = {
    AGENT: "AGENT",
    CUSTOMER: "CUSTOMER",
};
export const ParticipantTimerType = {
    DISCONNECT_NONCUSTOMER: "DISCONNECT_NONCUSTOMER",
    IDLE: "IDLE",
};
export const ParticipantTimerAction = {
    Unset: "Unset",
};
